sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -31,6 +31,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/urfave/cli/v2"
+	"k8s.io/utils/strings/slices"
```

Comment:
How much will this increase the size of the binary? I'd prefer if we don't include unnecessary packages if that means pulling down the entire source and compile it. (We can always copy what `slices` do in your function). @jpmcb, will the compiler strip out the functions that aren't called from the package when `host-ctr` is built?

Reply:
I don't believe it is increasing the binary size at all since it was already being pulled as an indirect dependency.

Reply:
> since it was already being pulled as an indirect dependency.

^ this. Go binaries are statically linked so if there is a dependency that then pull in other dependencies, we get all of those as well. Since we already pull `k8s.io/utils` as an indirect dependency through `k8s.io/cri-api`, there shouldn't be any change to our the binary size and it's deps. I'm not sure what the Go compiler is doing under the hood for optimizations, but if there are functions/apis we aren't calling into, it shouldn't also bring those along. The `k8s.io/utils` [library is pretty thin too,](https://github.com/kubernetes/utils/blob/master/go.mod) so I'm not worried about it bringing in dependencies we don't want (like different versions of the k8s client-go or some containerd library).

This would be worrisome if we were bringing in a _new_ dependency but since it was already a `indirect` dependency, it should be ok.